MultiplayerMod/Multiplayer/Objects/MultiplayerObjects.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System.Collections.Generic;
+using UnityEngine;
+
+namespace MultiplayerMod.Multiplayer.Objects;
+
+public class MultiplayerObjects {
+
+    private readonly Dictionary<long, GameObject> objects = new();
```

Comment:
Perhaps introduce type for our multiplayerId instead of long?

Reply:
Why? I don't see any benefit here (opposed to entities), could you elaborate more?

Reply:
Now it's `MultiplayerId`, because it requires more than just a simple numeric id. Probably we will combine these two into one for performance / networking reasons, but later.